ring_buffer/ring_buffer.py

```diff
@@ -1,11 +1,64 @@
 class RingBuffer:
-  def __init__(self, capacity):
-    self.capacity = capacity
-    self.current = 0
-    self.storage = [None]*capacity
+    def __init__(self, capacity):
+        self.capacity = capacity
+        self.current = 0
+        self.storage = [None]*capacity
 
-  def append(self, item):
-    pass
+    def append(self, item):
+         
+        if self.current >= self.capacity:
+          self.current = 0
+ 
+        self.storage[self.current] = item
+        self.current += 1
 
-  def get(self):
-    pass
\ No newline at end of file
+        # return self.storage
+
+    def get(self):
```

Comment:
Great job taking accountability for None values, however, this can still be simplified by using a list comprehension, this way you don't need to create any extra list and use any extra space.